component/Card/index.html

```diff
@@ -0,0 +1,264 @@
+          <img class="badge-icon" src="/component/Card/icon/close.png" alt="badge.png">
+        </div>
+      </div>
+    </div>
+    <!-- card with badge -->
+    <div class="card">
+      <div class="card-background">
+        <img
+        class="card-img"
+        src="https://www.planetware.com/wpimages/2020/01/india-in-pictures-beautiful-places-to-photograph-gateway-of-india-mumbai.jpg"
+        alt="img"
+      />
+          <div class="card-body">
+            <h3 class="card-title">Title</h3>
+            <small class="small-text">secondary text</small>
+            <p class="card-description">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Voluptatem quis, </p>
+          </div>
+        <img class="badge-icon" src="/component/Card/icon/like.png" alt="badge.png">
+      </div>
+    </div>
+    <!-- card with shadow -->
+
+    <div class="card ">
+      <div class="card-background shadow">
+        <img
+        class="card-img"
+        src="https://www.planetware.com/wpimages/2020/01/india-in-pictures-beautiful-places-to-photograph-gateway-of-india-mumbai.jpg"
+        alt="img"
+      />
+          <div class="card-body">
+            <h3 class="card-title">Title</h3>
+            <small class="small-text">secondary text</small>
+            <p class="card-description">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Voluptatem quis, </p>
+          </div>
+        <img class="badge-icon" src="/component/Card/icon/share.png" alt="badge.png">
+      </div>
+    </div>
+
+
+
+
+    <!-- basic card -->
+    <div class="card__container">
+      <div class="card__wrapper">
+        <div class="card_background">
+          <img
+          class="image"
+          src="https://www.planetware.com/wpimages/2020/01/india-in-pictures-beautiful-places-to-photograph-gateway-of-india-mumbai.jpg"
+          alt="img"
+        />
+        <div class="card__content">
+          <h3 class="title">Our Changing Planet</h3>
+          <p class="small_content">by Kurt Wagner</p>
+          <p class="description">
+            Visit ten places on our planet that are undergoing the biggest
+            changes today.
+          </p>
+        </div>
+        </div>
+       
+        <footer class="footer">
+          <button class="btn">READ</button>
+          <button class="btn">BOOKMARK</button>
+          <div class="icons">
+            <img class="icon" src="./icon/like.png" alt="" />
+            <img class="icon" src="./icon/share.png" alt="" />
+            <img class="icon" src="./icon/three-dot.png" alt="" />
+          </div>
+        </footer>
+      </div>
+      </div>
+      
+    
+<br>
+<!-- basic with text over media -->
+    <div class="card__container">
+      <div class="card__wrapper text_media">
+        <div class="card_background">
```

Comment:
Follow one style of giving css class names. If you are using double underscore, stick with it

Reply:
ok